### PR TITLE
Update QUEST_LV_0200.tsv

### PR DESCRIPTION
**Pull request by Requinn**
Reupload of pull request #662. Ommitted some changes to keep this PR smaller than last time. Sorry for the slight commit spam thing, haven't quite mastered branches...

Translated Lines:
000063
000358
005368
005381
006556 - 6599

Lines fixed for grammar/ellipsis/etc:
000021
000379
000387
000405
000409
000553
000576
000637
000934
001016-001018
001206
001208
001211

Names fixed:
The soldier,  ->  Soldier  (All Instances)
The commander, ->  Commander   (All instances)
Bilhemina ->  Wilhemina
Pilipas  -> Philip  (Not all instances)
Rudolpas  -> Rudolph  (Not all instances)
Aspas  -> Aspa   (Not all instances)
Mocolos/Mokolis/Micolas/etc  -> Mikolas 
Austeja Goddess  -> Goddess Austeja

Rail should be Lyle, Nomabis could be Norma.

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -172,7 +172,7 @@ QUEST_LV_0200_20150317_000171	There are many cursed documents and uncleaned obje
 QUEST_LV_0200_20150317_000172	Can you defeat them at the military district?{nl}If you could defeat them, we will go and capture them.
 QUEST_LV_0200_20150317_000173	The one who was caught last time told us that the stone statues were talking to him.{nl}Do you think that makes sense?
 QUEST_LV_0200_20150317_000174	I was able to arrest lots of thieves because of your help.{nl}Thank you.
-QUEST_LV_0200_20150317_000175	Guard Nomabis
+QUEST_LV_0200_20150317_000175	Guard Norma
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:175]: Why did you change this to Norma (노르마)? Should remain Nomabis.

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -816,13 +816,13 @@ QUEST_LV_0200_20150317_000815	You mean you returned him to the rear unit? {nl}Th
 QUEST_LV_0200_20150317_000816	Blood will be shed soon. {nl}I pity you for going in for Roy.
 QUEST_LV_0200_20150317_000817	So, just as I've heard. {nl}
 QUEST_LV_0200_20150317_000818	The next mission is to find the missing scouts. {nl}They were sent to Razysis Hills but have not returned for hours.
-QUEST_LV_0200_20150317_000819	The troops were attacked, and so was I. {nl}I hope we can at least find the report to send to sir Palipas..
+QUEST_LV_0200_20150317_000819	The troops were attacked, and so was I. {nl}I hope we can at least find the report to send to Sir Philip..
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:819]: Philip -> Philipas

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -816,13 +816,13 @@ QUEST_LV_0200_20150317_000815	You mean you returned him to the rear unit? {nl}Th
...
 QUEST_LV_0200_20150317_000820	Our squad is all dead because of my incompetence. {nl}I'm the only one alive but I don't think I have much time left either.
 QUEST_LV_0200_20150317_000821	The report is all soaked in blood and can't be read. {nl}I can't stand that my men's efforts were all for nothing.
 QUEST_LV_0200_20150317_000822	There is one last favor I would like to ask you. {nl}The remains of my men are in the hills. Please take care of their keepsakes.
 QUEST_LV_0200_20150317_000823	I feel sorry for them abandoned among the monsters.
 QUEST_LV_0200_20150317_000824	Thank you.{nl}My men can now rest in peace. {nl}
-QUEST_LV_0200_20150317_000825	Oh, I've found some parts that you can read from my men's reports. {nl}Please tell sir Philipas that the monsters will attack the troops soon.
+QUEST_LV_0200_20150317_000825	Oh, I've found some parts that you can read from my men's reports. {nl}Please tell Sir Philip that the monsters will attack the troops soon.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:825]: Philip -> Philipas

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -836,7 +836,7 @@ QUEST_LV_0200_20150317_000835	When will this pitiful feeling end. {nl}When I loo
 QUEST_LV_0200_20150317_000836	I can't help that the supplies or backup is delayed. {nl}But tellin us to just wait is too much.
 QUEST_LV_0200_20150317_000837	I'm afraid where the monsters will jump out from. {nl}I plan to run away if an unknown monster shows up.
 QUEST_LV_0200_20150317_000838	We're making up for the lacking supplies. {nl}If the supplies don't arrive, we have to cover for it on site.
-QUEST_LV_0200_20150317_000839	Somehow, at least sir Philipas' troops..
+QUEST_LV_0200_20150317_000839	Somehow, at least Sir Philip's troops..
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:839]: Philip -> Philipas

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -859,10 +859,10 @@ QUEST_LV_0200_20150317_000858	We are after an unknown monster. {nl}I hope it's n
 QUEST_LV_0200_20150317_000859	That's weird. I think I saw a Deadborn.. {nl}What do you think?
 QUEST_LV_0200_20150317_000860	Dear Revelator, please lend us your strength! {nl}We're fighting against Salister and us alone is not enough.
 QUEST_LV_0200_20150317_000861	You're great as expected! Commander Vacenin wants to meet you. Go to the camp quickly.
-QUEST_LV_0200_20150317_000862	Soldier Rudolpas
+QUEST_LV_0200_20150317_000862	Soldier Rudolph
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:862]: Rudolph -> Rudolfas

File: QUEST_LV_0200.tsv [hunk 6]
```diff
@@ -859,10 +859,10 @@ QUEST_LV_0200_20150317_000858	We are after an unknown monster. {nl}I hope it's n
...
 QUEST_LV_0200_20150317_000863	Even if you're the Revelator, this place is too dangerous. {nl}Please go to a safe area.
 QUEST_LV_0200_20150317_000864	What did I tell you. {nl}Thanks for your help but return quickly.
-QUEST_LV_0200_20150317_000865	Soldier Aspas
+QUEST_LV_0200_20150317_000865	Soldier Aspa
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:865]: Why did you change it to Aspa, while the Korean counterpart is Aspas?

File: QUEST_LV_0200.tsv [hunk 7]
```diff
@@ -2202,15 +2202,15 @@ QUEST_LV_0200_20150317_002201	Defeat Throneweaver
 QUEST_LV_0200_20150317_002202	Defeat Throneweaver for the sad owl
 QUEST_LV_0200_20150317_002203	Defeat Throneweaver that is destroying the owl sculptures. Talk to the sad owl again.
 QUEST_LV_0200_20150317_002204	Defeat Throneweaver
-QUEST_LV_0200_20150317_002205	Talk with the soldier, Rudolpas
-QUEST_LV_0200_20150317_002206	Talk with the soldier, Rudolpas
+QUEST_LV_0200_20150317_002205	Talk with Soldier Rudolpas
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2205]: Rudolpas -> Rudolfas

File: QUEST_LV_0200.tsv [hunk 8]
```diff
@@ -2202,15 +2202,15 @@ QUEST_LV_0200_20150317_002201	Defeat Throneweaver
...
+QUEST_LV_0200_20150317_002206	Talk with Soldier Rudolpas
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2206]: Rudolpas -> Rudolfas

File: QUEST_LV_0200.tsv [hunk 9]
```diff
@@ -2202,15 +2202,15 @@ QUEST_LV_0200_20150317_002201	Defeat Throneweaver
...
 QUEST_LV_0200_20150317_002207	Defeat the Meleech that suddenly appeared
-QUEST_LV_0200_20150317_002208	The Mellich appeared while you were talking to the soldier, Rudolpas. Defeat Mellich.
+QUEST_LV_0200_20150317_002208	The Mellich appeared while you were talking to Soldier Rudolpas. Defeat Mellich.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:2208]: Rudolpas -> Rudolfas

File: QUEST_LV_0200.tsv (hunk 1 above)
Comment (Requinn) [inline QUEST_LV_0200.tsv:175]: I was checking if nomabis was an actual name, like wilhelmina, and the apparent closest was just Norma. So I only changed one line to see if anyone would say something about it.

File: QUEST_LV_0200.tsv (hunk 2 above)
Comment (Requinn) [inline QUEST_LV_0200.tsv:819]: However, Philipas just kind of feels wrong despite its Korean spelling.

File: QUEST_LV_0200.tsv [hunk 10]
```diff
@@ -355,7 +355,7 @@ QUEST_LV_0200_20150317_000354	That's enough ingredients to make elixir of freedo
 QUEST_LV_0200_20150317_000355	I will give you this freedom elixir. {nl}Use it on petrified guards or grave thieves. {nl}Their souls trapped in the body might tell you something about what they last witnessed.
 QUEST_LV_0200_20150317_000356	The angry souls might attack you.
 QUEST_LV_0200_20150317_000357	There was nothing worth noting save for the victims being overwhelmed with fear before their deaths.{nl}I guess I better look for even older victims.
-QUEST_LV_0200_20150317_000358	
+QUEST_LV_0200_20150317_000358	Before they die, petfried victims release evil thoughts.{nl}Please use the incense and the stone while the knights are gone and I'll be heading over.{When enough of the evil thoughts have gathered, we can see what happened on that day.
```
Comment (vtange) [inline QUEST_LV_0200.tsv:358]: petrified

File: QUEST_LV_0200.tsv (hunk 1 above)
Comment (Sunaries) [inline QUEST_LV_0200.tsv:175]: > I was checking if nomabis was an actual name, like wilhelmina, and the apparent closest was just Norma. So I only changed one line to see if anyone would say something about it.

This is a translation project, not a localization project. Just because you've never heard of a name doesn't mean you should change it. Some NPCs contain names that are widely used in the world, and some of them contain names that are made up but belong to the world of Tree of Savior.

File: QUEST_LV_0200.tsv (hunk 2 above)
Comment (Sunaries) [inline QUEST_LV_0200.tsv:819]: See previous note. What is exactly so wrong about Philipas?

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.
